oap-server/server-core/src/test/resources/component-libraries.yml

```diff
@@ -266,6 +266,9 @@ InfluxDB:
 influxdb-java:
   id: 90
   languages: Java
+brpc-java:
```

Comment:
I think test file actually doesn't need to change every time.

Reply:
done